diagnostics/precip_buoy_diag/settings.jsonc

```diff
@@ -0,0 +1,69 @@
+// Convection Transition
+//
+// This requires sub-daily-timescale precipitation rate, air temperature, specific humidity, 
+// and surface pressure (optional).
+// For further documentation and user
+// options, see comments in convective_transition_diag_v1r3.py.
+// This also requires the following Python modules:
+// os, glob, json, cython, xarray, scipy, matplotlib, networkx, warnings,
+// numba.
+// The code is in Python (3.7).
+//
+// All text to the right of an unquoted "//" is a comment and ignored, as well
+// as blank lines (JSONC quasi-standard.) JSON doesn't support line continuation
+// in strings, so you'll want to turn word wrap on in your editor.
+//
+{
+	"settings": {
+		"driver": "precip_buoy_diag.py",
+		"long_name": "Precipitation Buoyancy Diagnostics",
+		"realm": "atmos",
+		"description": "Precipitation Buoyancy Diagnostics",
+		// threshold precipitation value to compute precipitating pdfs
+		"pod_env_vars": { "PRECIP_THRESHOLD": "0.25", 
+		// option to handle sigma co-ordinates. Change VERT_TYPE to "sigma"
+		"VERT_TYPE" : "pres"},
+		"runtime_requirements": {
+			"python3": ["numpy", "scipy", "cython", "matplotlib", "xarray", "numba", "networkx"]
+		}
+	},
+	"varlist": [{
```

Comment:
Thanks for converting to the new settings file format. Please add a [dimensions section](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_settings.html#dimensions-section) defining the `"lat"`, `"lon"`, `"lev"` and `"time"` coordinates referenced in the varlist section below.

Reply:
Added dimensions section in settings file b5efdcd